docs/Examples/watch-only-wallet.js

```diff
@@ -0,0 +1,81 @@
+'use strict';
+
+const bcoin = require('../..');
+
+(async () => {
+
+  // use well known test mnemonic
+  const phrase = [
```

Comment:
This could be on three lines and not go over 80 columns, and there is a trailing comma.